src/Extensions/Hosting.Services.Web/Middlewares/ActivityEnrichmentMiddleware.cs

```diff
@@ -11,16 +11,11 @@ namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
 	/// <summary>
 	/// Enrich request activity with Result, SubType and Metadata
 	/// </summary>
-	internal class ActivityEnrichmentMiddleware
+	internal class ActivityEnrichmentMiddleware : IMiddleware
 	{
-		public ActivityEnrichmentMiddleware(RequestDelegate next) => m_next = next;
-
-		/// <summary>
-		/// Invoke middleware
-		/// </summary>
-		public async Task InvokeAsync(HttpContext context)
+		async Task IMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
```

Comment:
I think it's more readable to leave the access mofidier in:
```suggestion
		private async Task IMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
```

Reply:
I don't think it will build like that, explicit interface implementation doesn't allow access modifier (as in interface)

Reply:
Interesting, ok then.